### PR TITLE
TapGestureRecognizer with closure

### PR DESCRIPTION
**Pull request by gmazzei**
## Summary ##

An extension for `UIView` that allows adding a `UITapGestureRecognizer` with a closure, instead of using a selector.

### CODE REVIEW

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Quality (Swiftlint): 96.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/tap-gesture-closure) for more details.

File: WolmoCoreTests/Extensions/UIKit/UIView/TapRecognizerSpec.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+
+import Quick
+import Nimble
+import WolmoCore
+
+public class TapRecognizerSpec: QuickSpec {
```
Comment (gmazzei) [inline WolmoCoreTests/Extensions/UIKit/UIView/TapRecognizerSpec.swift]: I've been looking how to test this properly but I couldn't find any suitable way to simulate a tap on Nimble.
Just [this](http://vojtastavik.com/2016/03/30/testing-gesture-recognizers/), but it didn't work. I ended up checking if the recognizer had been injected into the view, as the code below shows. What do you think?

Comment (DiegoQuiros94) [reply WolmoCoreTests/Extensions/UIKit/UIView/TapRecognizerSpec.swift]: Usually for UI extensions we don't add tests, it takes too much work to make UI tests, you can add an example in the WolmoCoreDemo target to test it out and show that it's working.

**Comment by gmazzei** [comment]
Okay, great!

**Comment by gmazzei** [comment]
I've added a demo 👍

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Coverage (Slather): 82.87
:white_check_mark: Code Quality (Swiftlint): 96.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-core-ios/branches/tap-gesture-closure) for more details.

**Pull request merged**